disco_aws_automation/disco_storage.py

```diff
@@ -195,16 +192,6 @@ def configure_storage(self,
         logging.debug("mapped %s to root partition", disk_names[current_disk])
         current_disk += 1
 
-        # Map the latest snapshot for this hostclass
-        if map_snapshot:
-            latest = self.get_latest_snapshot(hostclass)
-            if latest:
-                self.wait_for_snapshot(latest)
-                current_name = disk_names[current_disk]
-                bdm[current_name] = self.create_snapshot_bdm(latest, iops)
-                logging.debug("mapped %s to snapshot %s", current_name, latest.id)
-                current_disk += 1
-
         # Map extra disk
         if extra_disk:
```

Comment:
So EBS is always xvdz. but there is now possibility that extra disk will use xvdz if the instance type is hs1.8xlarge with an extra disk . see http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html